gapis/api/vulkan/state_rebuilder.go

```diff
@@ -2637,6 +2647,37 @@ func (sb *stateBuilder) createGraphicsPipeline(gp GraphicsPipelineObjectʳ) {
 			)).Ptr())
 	}
 
+	pipelineRasterizationState := NewVkPipelineRasterizationStateCreateInfoᶜᵖ(memory.Nullptr)
```

Comment:
just name it `rasterizationState` to stick with everything else that drops the "pipeline" part of it since, well, we're in a function called `createGraphicsPipeline`. It will also make it match the struct field names more closely.